homeassistant/components/dynalite/__init__.py

```diff
@@ -8,17 +22,63 @@
 
 # Loading the config flow file will register the flow
 from .bridge import DynaliteBridge
-from .config_flow import configured_hosts
-from .const import CONF_BRIDGES, DATA_CONFIGS, DOMAIN, LOGGER
+from .const import CONF_BRIDGES, CONF_NOWAIT, DATA_CONFIGS, DOMAIN, LOGGER
+
+
+def num_string(value):
+    """Test if value is a string of digits, aka an integer."""
+    new_value = str(value)
+    if new_value.isdigit():
+        return new_value
+    raise vol.Invalid("Not a string with numbers")
+
+
+CHANNEL_DATA_SCHEMA = vol.Schema(
+    vol.All(
+        {
+            vol.Optional(CONF_NAME): str,
+            vol.Optional(CONF_FADE): vol.Coerce(float),
+            vol.Optional(CONF_HIDDENENTITY, default=False): vol.Coerce(bool),
+        }
+    )
+)
+
+CHANNEL_SCHEMA = vol.Schema({num_string: vol.Any(CHANNEL_DATA_SCHEMA, None)})
+
+AREA_DATA_SCHEMA = vol.Schema(
+    vol.All(
+        {
+            vol.Required(CONF_NAME): str,
+            vol.Optional(CONF_FADE): vol.Coerce(float),
+            vol.Optional(CONF_CHANNEL): CHANNEL_SCHEMA,
+        },
+    )
+)
+
+AREA_SCHEMA = vol.Schema({num_string: vol.Any(AREA_DATA_SCHEMA, None)})
+
+PLATFORM_DEFAULTS_SCHEMA = vol.Schema({vol.Optional(CONF_FADE): vol.Coerce(float)})
+
+
+BRIDGE_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): str,
+        vol.Required(CONF_HOST): str,
+        vol.Optional(CONF_PORT, default=DEFAULT_PORT): int,
+        vol.Optional(CONF_LOGLEVEL): str,
```

Comment:
Log level should not be a config option. Logging level should be changed by using logging filters. This can be done with the `logger` integration in home assistant.

In the library each module should use a logger preferably using the module path:
```py
_LOGGER = logging.getLogger(__name__)
```